.github/workflows/require-allow-edits.yml

```diff
@@ -0,0 +1,14 @@
+name: Require “Allow Edits”
+
+on: [pull_request_target]
+
+jobs:
+  _:
```

Comment:
why `_`?

Reply:
The display on the PR will be `Require "Allow Edits" / _` with this. If there's a different name "after the slash" you'd prefer, I'm happy to change it.

Reply:
ohh hmm, looks like by providing the `name` field the `_` is overridden :-)

Reply:
If the `name` field is used I'm fine with it. I asked because I thought `_` had some implicit hidden behavior maybe 😅